apps/docs/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {}
```

Comment:
is leaving `rules` empty just setting no lint rules at all ?

Reply:
- it's for overrides, so does not affect base preset/rules
- it's autogenerated by Nx